modules/custom/invitation_manager/invitation_manager.module

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Contains invitation_manager.module.
+ */
+
+/**
+ * Implements hook_page_attachments().
+ * This function loads assets from the Service Canada GC Invitation Manager library
+ * Only public page sessions (not when a user is logged in) load survey pop-up
+ */
+function invitation_manager_page_attachments(array &$attachments) {
+  $node = \Drupal::request()->get('node');
+  if (is_object($node) && 
+    $node->getType() === 'landing_page' && 
+    \Drupal::currentUser()->isAuthenticated() === false) {
+
+    if (!in_array('core/jquery', $attachments['#attached']['library']))
```

Comment:
@RabiaSajjad Is this check for the jquery needed if `core/jquery` is already stated as a dependency in the libraries.yml?

Reply:
thanks for reviewing @JVickery-TBS. The invitation manager code needs jQuery to execute, so this is only to make sure that jQuery is loaded before invitation manger script starts executing